smokeTest.js

```diff
@@ -0,0 +1,112 @@
+const XETHRU_STATE = require('./SessionStateXethruEnum.js')
+const MOV_THRESHOLD = 17
+const IM21_DOOR_STATUS = require('./IM21DoorStatusEnum')
+const testLocation1Id = 'TestLocation1'
+const door_coreID = 'door_coreID'
+const radar_coreID = 'radar_coreID'
+const { helpers } = require('brave-alert-lib')
+
+function getRandomInt(min, max) {
+    min = Math.ceil(min)
+    max = Math.floor(max)
+    return Math.floor(Math.random() * (max - min) + min) //The maximum is exclusive and the minimum is inclusive
+}
+
+function getRandomArbitrary(min, max) {
+    return Math.random() * (max - min) + min
+}
+
+const destinationURL = process.argv[1];
+const recipientNumber = process.argv[2];
+const twilioNumber = process.argv[3];
+
+const axios = require('axios').default;
+const sleep = (millis) => new Promise(resolve => setTimeout(resolve, millis));
+axios.defaults.headers.post['Content-Type'] = 'application/x-www-form-urlencoded';
+axios.defaults.baseURL = destinationURL;
+      
+smokeTest(recipientNumber, twilioNumber);
+  
+async function smokeTest(recipientNumber, twilioNumber){  
+    await setup(recipientNumber, twilioNumber, testLocation1Id, door_coreID, radar_coreID)
+    for(let i = 0; i<15; i++){
+        await movement(testLocation1Id, getRandomInt(MOV_THRESHOLD + 1,100), getRandomInt(MOV_THRESHOLD + 1,100))
+    }
+    await im21Door(door_coreID, IM21_DOOR_STATUS.OPEN)
+    for(let i = 0; i<5; i++){
+        await silence(testLocation1Id, getRandomInt(0,MOV_THRESHOLD), getRandomInt(0,MOV_THRESHOLD))
+    }
+    await im21Door(door_coreID, IM21_DOOR_STATUS.CLOSED)
+    for(let i = 0; i<15; i++){
+        await movement(testLocation1Id, getRandomInt(0,MOV_THRESHOLD), getRandomInt(0,MOV_THRESHOLD))
+    }
+    for(let i = 0; i<15; i++){
+        await movement(testLocation1Id, getRandomInt(MOV_THRESHOLD + 1,100), getRandomInt(MOV_THRESHOLD + 1,100))
+    }
+    for(let i = 0; i<85; i++){
+        await silence(testLocation1Id)
+    }
+    await teardown()
```

Comment:
It would probably be a good idea to put the `teardown` in a `finally` block since we'll want to clear it out of the DB even if something else goes wrong.

Reply:
yes! I like this suggestion.